src/VisualStudio/Core/Def/PublicAPI.txt

```diff
@@ -1,6 +1,8 @@
 Microsoft.VisualStudio.LanguageServices.Progression.GraphNodeCreation
 Microsoft.VisualStudio.LanguageServices.VisualStudioWorkspace
 Microsoft.VisualStudio.LanguageServices.VisualStudioWorkspace.CreatePortableExecutableReference(string filePath, Microsoft.CodeAnalysis.MetadataReferenceProperties properties) -> Microsoft.CodeAnalysis.PortableExecutableReference
+Microsoft.VisualStudio.Shell.Interop.IComWrapperFactory
+Microsoft.VisualStudio.Shell.Interop.IComWrapperFactory.CreateAggregatedObject(object managedObject) -> object
```

Comment:
I think it's unavoidable for now. The compiler won't let you use TypeIdentityAttribute on a non-public type.
Eventually though, as the TODO comment in IComWrapper.cs says, it'll disappear when we use the real embedded interop type from Microsoft.VisualStudio.Shell.Interop.14.0.Design